.azure/gpu-tests-lite.yml

```
@@ -0,0 +1,126 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - "master"
+      - "release/*"
+      - "refs/tags/*"
```

Run only when these are changed

```suggestion
      - "refs/tags/*"
  paths:
    include:
      - ".azure/gpu-tests-lite.yml"
      - "requirements/lite/**"
      - "src/lightning_lite/**"
      - "tests/tests_lite/**"
```